applications/radiation_application/custom_elements/radiation_3d.cpp

```diff
@@ -351,7 +353,7 @@ namespace Kratos
 	msAux(3,3)+=N[3]*N[3];// + N[3]*N[0] + N[3]*N[1] + N[3]*N[2];
 	    
 	    
-	//	    noalias(rLeftHandSideMatrix) += 1.0 * Weight * absorptioncoefficient * msAux; //LO COMENTO AHORA
+	noalias(rLeftHandSideMatrix) += 1.0 * Weight * absorptioncoefficient * msAux; //LO COMENTO AHORA
```

Comment:
the comment is in spanish, (please do not use spanish in the code). Also i think it is outdated